core-bundle/src/Controller/BackendPreviewSwitchController.php

```diff
@@ -169,7 +169,7 @@ private function getMembersDataList(BackendUser $user, Request $request): array
             WHERE
                 username LIKE ? $andWhereGroups
                 AND login=1
-                AND disable!=1
+                AND disable=0
```

Comment:
```suggestion
                AND !disable
```
How about using boolean expressions?

Reply:
Does this work in all MySQL versions?

Reply:
Yes (tested with 5.5 and up)
https://www.db-fiddle.com/f/66jye1UoJzVkL52qdrySne/0

Reply:
Like this? 60c0f0f1ba7bfa87a54c4b7324f0087b5ca7764c

Would this also work for `login AND !disable AND published` instead of `login=1 AND !disable AND published=1`?

Reply:
Yes. To me this would be the nicest syntax, not sure if it is the fastest though.

Reply:
It seems for performance reasons we should use `=0` and `<>0` 😕
https://stackoverflow.com/a/52073119/1031606
(But we should verify this if it is acutally the case in mariadb or mysql 8)

Reply:
Veryfied: https://www.db-fiddle.com/f/sskh7B7ZRiRCSeM92NZUep/1
We should use `=1` and `=0` for best performance.

So my suggestion needs to be reverted (sorry 🙈).

Reply:
No biggie. I have reverted the changes.